test/Project.toml

```diff
@@ -0,0 +1,6 @@
+[deps]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Trixi = "a7f1ee26-1774-49b1-8366-f1abc58fbfcb"
```

Comment:
If not using Trixi specifically as a module in the tests, there's no need to add it as a dependency here
```suggestion
```

Reply:
Thanks!